master/docs/manual/cfg-buildsteps.rst

```diff
@@ -2443,6 +2443,14 @@ Hyperlinks are added to the build detail web pages for each triggered build.
 
         It is possible, but not advisable, to create a cycle where a build continually triggers itself, because the schedulers are specified by name.
 
+``unimportantSchedulerNames``
+    When ``waitForFinish`` is ``True``, all schedulers in this list will not cause the trigger step to fail. unimportantSchedulerNames must be a subset of schedulerNames
+    If ``waitForFinish`` is ``False``, unimportantSchedulerNames will simply be ignored.
+    
+.. note::
+
+        ``schedulerNames`` must have at least one scheduler not present in ``unimportantSchedulerNames``
```

Comment:
that also means that the following code (and its test) must go

 if len(schedulerNames) <= len(unimportantSchedulerNames):
         config.error(
                 "SchedulerNames must at least contain an important scheduler"
         )